contrib/systemd/cilium-health

```diff
@@ -0,0 +1 @@
+CILIUM_HEALTH_OPTS="-d --prometheus-serve-addr=:9091"
```

Comment:
I'm wondering whether we should include Prometheus support by default or not. If not, then maybe we should provide some env variable for Vagrant, i.e. `PROMETHEUS` - if enabled, then our vagrant scripts should add `--prometheus-serve-addr` to sysconfig files.

@aanm @ianvernon @eloycoto WDYT?